Src/VimWpf/Implementation/CommandMargin/CommandMarginController.cs

```diff
@@ -333,7 +333,7 @@ private void UpdateCommandLine(string commandLine)
         /// </summary>
         private bool HandleHistoryNavigation(KeyInput keyInput)
         {
-            var handled = _vimBuffer.Process(KeyInputUtil.VimKeyToKeyInput(VimKey.Up)).IsAnyHandled;
+            var handled = _vimBuffer.Process(keyInput).IsAnyHandled;

```

Comment:
This looks like it was accidental and the intent was to use the supplied `keyInput` for the processing.